test/unit/base_cli_test.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+# copyright: 2017, Chef Software Inc.
+
+require 'helper'
+require 'thor'
+
+describe 'BaseCLI' do
+  let(:cli) { Inspec::BaseCLI.new }
+
+  describe 'merge_options' do
+    it 'json options override cli' do
+      json = { "backend_cache":true }
```

Comment:
`read_config` doesn't return JSON but rather parsed JSON. Can we change the variable name to `parsed_json` so it's clear? initially I thought you were trying to represent actual JSON here and thought you forgot the quotes around the whole string. 🙂 

Also, this Hash has a key of a symbol named `:backend_cache` with the way you've written it. `JSON.parse` won't return it like that, so we should probably change this to:

```ruby
parsed_json = { 'backend_cache' => true }
```